### DISCUSSION TITLE
npx init doesn't populate globals.css & tailwind.config.js anymore

### DISCUSSION THREAD

**Discussion by YugandharrPatil** [shadcn-ui/ui#997] [Q&A]
Just an hour ago, I ran npx init in my fresh next app (with src) and it populated my globals.css and tailwind config properly with the theme colors. However, just now when I run it in a fresh next app (again with src), it doesn't populate them my globals.css at all and my tailwind config partially. Am I the only one facing this? It there a solution?

**Comment by abd-ulbasit** [comment]
I am having the same issue. I am trying to build an app with t3 and init-ing shadcn/ui is now working properly

**Comment by abd-ulbasit** [comment]
but doing it manually, works for me.
